### PR TITLE
Instrument visualization

### PR DESCRIPTION
**Pull request by cquiroz**
This is an initial demo version of visualization. Note that several properties are hardcoded and only some of the features are actually working but it is a first step.

### CODE REVIEW

File: explore/src/main/scala/explore/tabs/TargetTabContents.scala [hunk 1]
```diff
@@ -19,35 +18,15 @@ import japgolly.scalajs.react.raw.JsNumber
...
-      (BreakpointName.md, (996, 10, layoutMd))
-      // (BreakpointName.sm, (768, 8, layout)),
-      // (BreakpointName.xs, (480, 6, layout))
-    )
-
   type Props = View[Option[Focused]]
 
   final case class State(treeWidth: JsNumber)
 
+  implicit val propsReuse: Reusability[Props] = Reusability.by(_.get)
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/tabs/TargetTabContents.scala:29]: You should get this for free when importing `crystal.react.implicits_`.

File: explore/src/main/scala/explore/Tile.scala [hunk 2]
```diff
@@ -9,29 +9,36 @@ import japgolly.scalajs.react.vdom.html_<^._
 import react.common._
 import react.semanticui.collections.menu._
 
-final case class Tile(title: String) extends ReactPropsWithChildren[Tile](Tile.component)
+final case class Tile(title: String, movable: Boolean)
```
Comment (rpiaggio) [inline explore/src/main/scala/explore/Tile.scala:12]: Nice

File: model/src/main/scala/explore/model/enum/Display.scala [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+import cats.data.NonEmptyList
+import gem.util.Enumerated
+import monocle.Iso
+
+/**
+  * Enum to indicate visibility of an item, roughly equivalent to css display
+  */
+sealed trait Display extends Product with Serializable {
+  def fold[A](hidden: => A, inline: => A): A =
```
Comment (rpiaggio) [inline model/src/main/scala/explore/model/enum/Display.scala:14]: Nice

Comment (cquiroz) [reply model/src/main/scala/explore/model/enum/Display.scala:14]: Learned from https://www.youtube.com/watch?v=cS4MOUsuWYc

**Pull request merged**